x-pack/test/api_integration/apis/search/search.ts

```diff
@@ -109,25 +109,6 @@ export default function ({ getService }: FtrProviderContext) {
         verifyErrorResponse(resp.body, 400, 'Request must contain a kbn-xsrf header.');
       });
 
-      it('should return 400 when unknown index type is provided', async () => {
```

Comment:
I would replace this test with calling the `rollup` api with a non-rollup index

Reply:
👍